eth/ethconfig/config.go

```diff
@@ -53,6 +53,7 @@ var Defaults = Config{
 	NetworkId:          0, // enable auto configuration of networkID == chainID
 	TxLookupLimit:      2350000,
 	TransactionHistory: 2350000,
+	LogHistory:         2350000,
```

Comment:
did you run `go run github.com/fjl/gencodec -type Config -formats toml -out gen_config.go`?